### PR TITLE
Term definitions for workflows

### PR DESCRIPTION
**Pull request by nissimsan**
This adds LD terms for Workflows. Also makes type mandatory. 

Note that the BPMN terms are not resolving. I looked extensively for an authoritative vocab for these definitions, but without luck. Going with `https://w3id.org/traceability#...` might be preferable? Change requests welcome!

### CODE REVIEW

**Comment by nissimsan** [comment]
Shout out to @VladimirAlexiev - thank you for the assistance!

**Comment by OR13** [comment]
Duplicate of https://github.com/w3c-ccg/traceability-vocab/pull/815/files

File: docs/openapi/components/schemas/common/Workflow.yml [hunk 1]
```diff
@@ -7,18 +7,22 @@ properties:
...
+      term: instance
+      '@id': https://w3id.org/traceability#instance
 additionalProperties: false  
-required:
-  - instance
 example: |-
   {
     "definition": [
-      "https://w3id.org/traceability/#sima-steel-license"
+      "https://w3id.org/traceability/#us-cbp-entry"
```
Comment (OR13) [inline docs/openapi/components/schemas/common/Workflow.yml:25]: this looks good

File: docs/openapi/components/schemas/common/Workflow.yml [hunk 2]
```diff
@@ -7,18 +7,22 @@ properties:
     type: array
     items:
       type: string
+    $linkedData:
+      term: definition
+      '@id': https://w3id.org/traceability#definition
```
Comment (OR13) [inline docs/openapi/components/schemas/common/Workflow.yml:12]:
```suggestion
      '@id': https://w3id.org/traceability#workflow-definition
```

File: docs/openapi/components/schemas/common/Workflow.yml [hunk 3]
```diff
@@ -7,18 +7,22 @@ properties:
...
   instance:
     description: The Workflow Instance identifies this particular workflow. Subsequent presentations may indicate correlation by referencing the same instance identifier.
     type: array
     items:
       type: string
+    $linkedData:
+      term: instance
+      '@id': https://w3id.org/traceability#instance
```
Comment (OR13) [inline docs/openapi/components/schemas/common/Workflow.yml:20]:
```suggestion
      '@id': https://w3id.org/traceability#workflow-instance
```

**Review by mkhraisha — APPROVED** [review]

**Comment by nissimsan** [comment]
@nissimsan  to revisit

**Comment by BenjaminMoe** [comment]
Waiting for @nissimsan

**Review by brownoxford — APPROVED** [review]

**Pull request merged**